src/sparseml/pytorch/utils/exporter.py

```diff
@@ -53,8 +54,15 @@
     "export_onnx",
 ]
 
+_PARSED_TORCH_VERSION = version.parse(torch.__version__)
 
-DEFAULT_ONNX_OPSET = 9 if torch.__version__ < "1.3" else 11
+DEFAULT_ONNX_OPSET = (
+    9
+    if _PARSED_TORCH_VERSION < version.parse("1.3")
+    else 10
```

Comment:
why 10? We should be using at least 11 to get quantization support for torch 1.5+

Reply:
Oh yes great catch ty!